docs/api/content-tracing.md

```diff
@@ -145,7 +150,7 @@ TraceBufferUsage value is determined, the `callback` is called.
 `callback` will will be called every time the given event occurs on any
 process.
 
-## tracing.cancelWatchEvent()
+### `tracing.cancelWatchEvent()`
 
-Cancel the watch event. If tracing is enabled, this may race with the watch
+Cancel the watch event. If tracing is enabled this may race with the watch
```

Comment:
"this may race" is a bit confusing. Is this accurate:

```
This may lead to a race condition with the watch event callback if tracing is enabled.
```